torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -103,25 +103,18 @@ class ShardingStrategy(Enum):
                    GPU memory until backward computation is done. It inserts reduce_scater
                    after backward computation for synchronizing and sharding gradients.
                    Sharded optimizer states are updated locally.
-    SHARD_OP(future support): Shard optimizer states only, this algorithm inserts all_gather
```

Comment:
No more support for shard_op is planned?

Reply:
yes, no need this, SHARD_OP_GRAD is better than SHARD_OP, also it is not feasible to implement SHARD_OP